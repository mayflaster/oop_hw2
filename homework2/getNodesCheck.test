
CreateGraph A
ListNodes A

CreateNode n1 10
AddNode A n1
ListNodes A

CreateNode n2 20
AddNode A n2
ListNodes A 

CreateNode n3 30
AddEdge A n2 n3 
ListNodes A 



CreateNode n4 40
CreateNode n5 50
AddEdge A n4 n5
CreateNode n1 30
ListNodes A

