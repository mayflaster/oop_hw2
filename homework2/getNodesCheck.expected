
created graph A
A contains:

created node n1 with cost 10
added node n1 to A
A contains: n1

created node n2 with cost 20
added node n2 to A
A contains: n1 n2

created node n3 with cost 30
added node n3 to A
added edge from n2 to n3 in A
A contains: n1 n2 n3


created node n4 with cost 40
created node n5 with cost 50
added node n4 to A
added node n5 to A
added edge from n4 to n5 in A
A contains: n1 n2 n3 n4 n5